crates/tokio_helper/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+use once_cell::sync::Lazy;
+use tokio::runtime::Runtime;
+
+pub static TOKIO: Lazy<Runtime> = Lazy::new(new_runtime);
+
+/// Instantiate a new runtime.
+pub fn new_runtime() -> Runtime {
```

Comment:
does this need to be `pub`? - I'd personally vote for only using the TOKIO static - we don't actually want more than one runtime.